cameraTrackingAPI/camera_calibrate.py

```diff
@@ -1,31 +0,0 @@
-import numpy as np
```

Comment:
No need for this anymore because all you have to do is call colorCalibrate to being that whole process..